gamemodes/terrortown/gamemode/client/cl_scoring.lua

```diff
@@ -138,6 +138,10 @@ function CLSCORE:CreatePanel()
 		self:HidePanel()
 	end)
 
+	frame.IsBlockingBindings = function(slf)
+		return self:IsBlockingBindings()
```

Comment:
not sure about this one, why is there the function parameter `slf` if you access `self` in the line below?

Reply:
Well the slf is frame specific, while the self is for CLSCORE.
I generally am used to write all arguments even if you dont use them.

Reply:
ahh, imho is is confusing, but this is fine for me as well